apps/ntag/ntagCmdApp/ntag_main.cpp

```diff
@@ -74,6 +74,13 @@ void PushNdef(void) {
     char buf[BUFSIZ], buf2[BUFSIZ];
 
     ntagAdapter.begin();
+
```

Comment:
The 4 lines below can be replaced with:
`dhubIO_PushBoolean(TAG_FORMATTED, DHUBIO_NOW, !ntagAdapter.isUnformatted());`